.github/MIGRATION.md

```diff
@@ -14,6 +20,131 @@
     - [Using blockstack.js](#using-blockstackjs-2)
     - [Using @stacks/encryption or @stacks/auth](#using-stacksencryption-or-stacksauth)
 
+## Stacks.js (&gt;=5.x.x) → (7.x.x)
+
+### Breaking Changes
+
+- The `@stacks/network` `new StacksNetwork()` objects were removed. Instead `@stacks/network` now exports objects like `STACKS_MAINNET`, which are static (and shouldn't be changed for most use-cases). [Read more...](#stacks-network)
+- The `ClarityType` enum was replaced by a readable version. The previous (wire format compatible) enum is still available as `ClarityWireType`. [Read more...](#clarity-representation)
+
+### Stacks Network
+
+From now on "network" objects are static (aka constants) and don't require instantiation.
+
+The `@stacks/network` package exports the following network objects:
+
+- `STACKS_MAINNET`
+- `STACKS_TESTNET`
+- `STACKS_MOCKNET`
+- `STACKS_DEVNET` (alias for `STACKS_MOCKNET`)
```

Comment:
```suggestion
- `STACKS_DEVNET`
- `STACKS_MOCKNET` (alias for `STACKS_DEVNET`)
```

We try to normalize the usage of devnet (and simnet), could this be presented as if `DEVNET` was the "real one" (even though it's not really the case)

Reply:
Happy to match it to what clarinet uses. I can leave our MOCKNET or rename to SIMNET -- not sure I follow completely.